RawSpeed/DngDecoder.cpp

```diff
@@ -81,6 +85,12 @@ RawImage DngDecoder::decodeRawInternal() {
   if (raw->hasEntry(SAMPLEFORMAT))
     sample_format = raw->getEntry(SAMPLEFORMAT)->getInt();
 
+  int compression = -1;
```

Comment:
Now that i *actually* look, we have already did `int compression = (*i)->getEntry(COMPRESSION)->getShort();` before. So just do
```c++
const int compression = raw->getEntry(COMPRESSION)->getShort();;
```
Sorry for asking about this earlier.